README.md

```diff
@@ -1,57 +1,44 @@
-    2. [must] Send a confirmation email to the given address when the Booking is saved
-    3. [optional] Create a validation to allow only email addresses
-    4. [optional] Use a ruby gem to automatically open the sent email during development (https://www.ruby-toolbox.com/categories/EMail_Fake_Servers)
-    5. [optional] Write automated tests
-3. Style the UI of the app!
-    1. [must] Use any front-end tools to make this application a bit nicer
-4. Create a statistics page
-    1. [must] Create a new route and a new controller for the statistics
-    2. [must] Display the summary of the booked days
-    3. [optional] Display the monthly summary
-    4. [optional] Write automated tests
-5. Add pagination into the /bookings page using the pagy ruby gem
-    1. [must] Add pagination to the bookings list. Every page should display 10 bookings
-    2. [optional] Use one of the ruby gems to achieve it (https://www.ruby-toolbox.com/categories/pagination)
-    3. [optional] Write automated tests
-6. Deploy this application to Heroku
-    1. [must] Deploy this simple application to Heroku and use their Postgresql database in production (https://devcenter.heroku.com/articles/getting-started-with-rails6)
-    2. [optional] Add addons for log handling, error monitoring, etc
-    
-    
-See you in two weeks!
+# Dear candidate!
+
+At this stage of the interview process, you'll be introduced to a very simple Ruby on Rails application, on which you'll have to do a number of exercises. Our goal with this is to examine the various skillsets you have. Let me just point that out, that you do NOT need to finish all the exercises in order to succeed. We'd rather like to learn the way you think, solve problems and your general knowledge on the topic by watching you work, than to just discuss the sheer number of LoC you wrote.
+
+The application you'll be working with today is practically a "guesthouse management app". You can book for a specific date, which will then be displayed on the main page. You're encouraged to try out it's functionality before starting any of the exercises.
+
+If you come across an obstacle which proves to be difficult, feel encouraged to ask us anything! We'll be glad to answer or ask some leading questions. (Or just utilize Google, StackOverflow or https://guides.rubyonrails.org/ to your liking.)
+
+I wish you success!
+
+## Let's begin!
+0. Run the server
+    1. open up https://paiza.cloud/en/ in your browser
+    2. sign in with your google account or create on with an email
+    3. create a new server: { name: "book-my-house", Web application development: "Ruby on Rails", Database: "PostgreSQL" }
+    4. open the menu in the to left corner of the screen and select 'git'>>'clone' and then this project repo
+    5. open a terminal window
+        - run these commands from /home/ubuntu (you might already be there, so it's worth cheking with 'echo $PWD')
+        - 'gem install bundler'
+        - 'bundle install'

```

Comment:
Nekem 2.7.0-ás rubyt rakott fel alapból, és így ez nem fut le. Lehetőségek:
- húzzuk fel a ruby verziót a repóban
- állítsuk be a cloudon a megfelelő ruby verziót, ha lehet ilyet